.travis.yml

```diff
@@ -25,6 +25,7 @@ addons:
 install:
   - npm install -g istanbul mocha@3
   - npm install
+  - npm insatll --save-dev
```

Comment:
There is a typing mistake here. It should be `install` instead of `insatll`

Reply:
My bad @geekyd :sweat_smile: